server/routes/authRoute.js

```diff
@@ -0,0 +1,31 @@
+const express = require("express");
+const router = new express.Router();
+
+const User = require("../models/User");
+
+// User signup route
+router.post("/register", async (req, res) => {
+	const user = new User(req.body);
```

Comment:
you may want to do some validation rather than trust the entire `body` that it sent over. 

For example, making sure email is valid, firstname and lastname are strings, etc.

Reply:
I added validation inside the User model, my bad I forgot to mention that in the feature description